.kitchen.yml

```diff
@@ -60,6 +64,31 @@ suites:
           checkpoint: true
           switch_user: yes
 
+- name: attributes
```

Comment:
This is similar enough to the default suite that I think it would make sense to use YAML anchors and aliases to avoid so much duplication. You should be able to add an anchor to the default YAML node, copy subnodes in here from an alias, and then just override the subnodes that are different.